scripts/python/run_spec_plan.py

```
@@ -0,0 +1,1302 @@
+    parser.add_argument("--query", default=None, help="Custom feature prompt if not using a ticket ID")
+    parser.add_argument("--tickets-file", type=Path, default=Path("docs/planning/TICKETS.md"), help="Path to TICKETS.md")
+    parser.add_argument("--include", default=DEFAULT_INCLUDE_PATTERNS, help="Glob patterns to pack via repomix")
+    parser.add_argument("--model", default=DEFAULT_MODEL, help=f"Model name (default: {DEFAULT_MODEL})")
+    parser.add_argument("--base-url", default=None, help=f"Custom API base URL (default: {DEFAULT_LOCAL_BASE_URL})")
+    parser.add_argument("--api-key", default=None, help="API Key")
+    parser.add_argument("--direct-nvidia", action="store_true", help="Connect directly to NVIDIA API")
+    parser.add_argument("--compress", action="store_true", help="Enable repomix compression")
+    parser.add_argument("--thinking", default="max", choices=["off", "low", "high", "max"], help="Thinking/reasoning effort level (default: max)")
+    parser.add_argument("--output-dir", type=Path, default=None, help="Custom plan directory path under ai/plans/")
+    parser.add_argument("--sprint", default=None, help="Sprint identifier (e.g. '0', '1'); nests the plan directory under ai/plans/sprint_<value>/")
+    parser.add_argument("--repomix-output", type=Path, default=Path("/tmp/kottab_spec_plan_repomix.xml"))
```

Ruff (0.16.3)

[error] Probable insecure usage of temporary file or directory: "/tmp/kottab_spec_plan_repomix.xml"

(S108)